src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_pricing.html.twig

```diff
@@ -11,19 +11,20 @@
                 </tr>
             </thead>
             <tbody>
-            {% for channelPricing in product.variants.first.channelPricings %}
-                <tr>
-                    <td class="five wide gray text">
-                        <strong>{{ channelPricing.channelCode|sylius_channel_name }} </strong>
-                    </td>
-                    <td>{{ money.format(channelPricing.price, product.channels.first.baseCurrency) }}</td>
-                    {% if channelPricing.originalPrice != null %}
-                        <td>{{ money.format(channelPricing.originalPrice, product.channels.first.baseCurrency) }}</td>
-                    {% else %}
-                        <td>-</td>
-                    {% endif %}
-                </tr>
-            {% endfor %}
+                {% set currencies = sylius_channels_currencies() %}
+                {% for channelPricing in product.variants.first.channelPricings %}
+                    {% set channel = channelPricing.channelCode|sylius_channel %}
+                    <tr>
+                        <td>{{ channel.name }}</td>
+                        {% set channelCode = channel.code %}
```

Comment:
I'm not sure if this variable is needed, as it is used in 2 places and similar to `channel.code`